src/payment/payment-methods.mock.ts

```diff
@@ -463,6 +463,7 @@ export function getStripeV3(method: string = 'card'): PaymentMethod {
         },
         initializationData: {
             stripePublishableKey: 'key',
+            cardFormIndividualElements,
```

Comment:
🏆  for booleans, a verb is preferred rather than a noun.

Maybe something like `shouldUseCardIndividualElements`